assets/app.js

```diff
@@ -154,6 +154,7 @@ new Vue({
         Multiselect: window.VueMultiselect.default
     },
     data: {
+        accessRight: AccessRight,
```

Comment:
use 2 variables instead: 
- `isAdmin: true/false` and use `v-if="isAdmin"` as condition in the template
- get `AppLanguages` via php from the decoded json from `descriptor: {'is_admin':true/false, 'languages':[ {'id':'de', 'status': true, ...},{....}, ... ] }` 
where `language.id` is the unique language code, `language.status` describes if the language is active by default. `language.flag` should NOT be encoded in the access rights, this is a static css class which renders the language flag icon, but rather set when we set `AppLanguages`